translations/strings.yaml

```diff
@@ -63,3 +63,22 @@ settingsDev:
   useStagingTitle: Use Staging Servers
   useStagingSubtitle: Staging services are used to test features before they go live to production
   restartRequired: VPN must be restarted to take effect
+
+splittunnel:
+  missingAppAction:
```

Comment:
It would be great if we could pick a more semantic name here. Is it a button or something else?

Reply:
Good catch, it's a call-to-action on a notification. Updated the name to "missingAppActionButton" as well as the comment to reflect that